### PR TITLE
Add release notes for Docker Engine 20.10.7

### PR DESCRIPTION
**Pull request by thaJeztah**

### CODE REVIEW

**Comment by thaJeztah** [comment]
@cpuguy83 @tonistiigi @tianon  ptal

**Review by cpuguy83 — APPROVED** [review]
LGTM

File: engine/release-notes/index.md [hunk 1]
```diff
@@ -22,6 +22,63 @@ for Docker Engine.
...
+    * Fix build-cache not being invalidated when using mounts [moby/buildkit#2076](https://github.com/moby/buildkit/pull/2076).
+* Fix build failures when `FROM` image is not cached when using legacy schema 1 images [moby/moby#42382](https://github.com/moby/moby/pull/42382).
+
+### Logging
+
+* Update the hcsshim SDK to make daemon logs on Windows less verbose [moby/moby#42292](https://github.com/moby/moby/pull/42292).
+
+### Rootless
+
+* Fix capabilities not being honoured when an image was built on a daemon with
```
Comment (samuelkarp) [inline engine/release-notes/index.md:56]: nit: I think we're using American English? :smile: 

```suggestion
* Fix capabilities not being honored when an image was built on a daemon with
```

File: engine/release-notes/index.md [hunk 2]
```diff
@@ -22,6 +22,63 @@ for Docker Engine.
...
+  with the `slirp4netns` port driver [moby/moby#42294](https://github.com/moby/moby/pull/42294).
+* Fix "Device or resource busy" errors when running docker-in-docker on a rootless
+  daemon [moby/moby#42342](https://github.com/moby/moby/pull/42342).
+
+### Packaging
+
+* Update containerd to v1.4.6, runc v1.0.0-rc95 to address [CVE-2021-30465](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-30465)
+  [moby/moby#42398](https://github.com/moby/moby/pull/42398), [moby/moby#42395](https://github.com/moby/moby/pull/42395),
+  [ocker/containerd-packaging#234](https://github.com/docker/containerd-packaging/pull/234)
+* Update containerd to v1.4.5, runc v1.0.0-rc94 [moby/moby#42372](https://github.com/moby/moby/pull/42372),
```
Comment (samuelkarp) [inline engine/release-notes/index.md:77]: Do we want to include this entry as well?  containerd 1.4.5 is replaced by 1.4.6 and runc 1.0.0-rc94 is replaced by 1.0.0-rc95.

Comment (thaJeztah) [reply engine/release-notes/index.md:77]: I was a bit in doubt; reason I included them both was that the v1.4.5 update (and runc 1.0.0-rc94) has a much larger changelog; I thought mentioning it (and providing links to the pull requests) could make those changes more discoverable.

File: engine/release-notes/index.md (hunk 1 above)
Comment (thaJeztah) [reply engine/release-notes/index.md]: Thx! Fixed 👍

File: engine/release-notes/index.md (hunk 2 above)
Comment (samuelkarp) [reply engine/release-notes/index.md:77]: We could write this in one entry with all the PRs?  Or we could leave it as two separate entries.  Either way seems fine to me.

**Review by samuelkarp — APPROVED** [review]
LGTM

**Review by tianon — APPROVED** [review]

**Review by tonistiigi — APPROVED** [review]
SGTM

**Comment by thaJeztah** [comment]
https://github.com/moby/moby/releases/tag/v20.10.7

**Pull request merged**